common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -362,17 +362,6 @@ d_rustbolt = {
 	}
```

Comment:
This could be a bit higher and use more space.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122812326-98077700-d2e2-11eb-95a5-17f5a5df4ca2.png)

</details>